ptn/boozebot/botcommands/DatabaseInteraction.py

```diff
@@ -336,18 +311,16 @@ def _update_db(self):
                 updated_db = True
                 print("Added carrier to the database")
 
-                # Post the notification to the Admin channel
-
                 embed = discord.Embed(title=f"New WineCarrier signed up!")
                 embed.add_field(
-                    name=f"Owner: {carrier.discord_username}: {carrier.carrier_name} ({carrier.carrier_identifier})",
-                    value=f"{carrier.wine_total // carrier.run_count} tonnes of wine on {carrier.platform}",
+                    name=f"Owner: {carrier_data.discord_username}: {carrier_data.carrier_name} ({carrier_data.carrier_identifier})",
+                    value=f"{carrier_data.wine_total // carrier_data.run_count} tonnes of wine on {carrier_data.platform}",
                     inline=False,
                 )
                 new_signups.append(embed)
-
-        print(all_carrier_ids_sheet)
-
+ 
+        print(f"all_carrier_sheet_ids: {[id for id in all_carrier_ids_sheet]}")
```

Comment:
```suggestion
        print(f"all_carrier_sheet_ids: {all_carrier_ids_sheet}")
```

`all_carrier_ids_sheet` is already a list of carrier IDs